docker/subset-fastq-dataset/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+tag=$1
+
+if [ -z $tag ]; then
```

Comment:
The tag here is the docker image tag. Not sure why it would be in hts specs. This is a convention we started with Rex because previously the image building was done manually that was very error prone and it was also rather difficult to find the correct tag for each image.